packages/fiori/src/DynamicPage.ts

```diff
@@ -278,6 +279,10 @@ class DynamicPage extends UI5Element {
 		return this._headerSnapped;
 	}
 
+	get hasSnappedTitleOnMobile() {
```

Comment:
Is it necessary to repeat this check?

Can't we just check this.dynamicPageTitle?._hasSnappedTitleOnMobile ?

Reply:
I’ve made changes so that we no longer need to repeat this check. Now, we send information from the parent to the child component due to the rendering order. This ensures we receive the correct state value.